requirements.txt

```
@@ -1,3 +1,4 @@
+torch==1.0
```

```suggestion
torch>=1.0
```

Greater than 1.0 should work also.  Thank you for the PR!